controllers/chat/sendSignedMessage.js

```diff
@@ -114,7 +114,7 @@ const sendSignedMesage = async (req, res) => {
 
     // Send PN to all members in group chat except sender
     if (channelType === 1) {
-      let better_channel_member = createdChannel?.channel?.better_channel_member;
+      let better_channel_member = createdChannel?.members;
       channelMember = better_channel_member.filter((element) => element.user_id !== req.userId);
       senderInfo = better_channel_member.find((element) => element.user_id === req.userId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change seems to be accessing the `members` property directly from `createdChannel`, instead of accessing it through `channel`. However, there's no null check for `createdChannel` before accessing its `members` property. This could potentially lead to a TypeError if `createdChannel` is undefined or null.
```diff
- let better_channel_member = createdChannel?.members;
+ let better_channel_member = createdChannel ? createdChannel.members : [];
```
This will ensure that `better_channel_member` is always an array, even when `createdChannel` is null or undefined, preventing potential TypeErrors when calling array methods on it.


<!-- This is an auto-generated comment by OSS CodeRabbit -->